docs/user_guide/dask.rst

```diff
@@ -0,0 +1,205 @@
+.. _dask:
+
+Lazy Computation with Dask
+==========================
+
+`Dask <https://dask.org/>`__ is integrated into Metagraph to provide lazy computation as well as lazy loading of data.
```

Comment:
Same here.  Can we change this to "... is integrated into Metagraph to allow algorithms and translations to be run in Dask task graphs, and to accept Dask data structures as input."